### ISSUE TITLE
Sessions not getting marked as crashed

### ISSUE THREAD

**Issue by gid-sentry** [getsentry/sentry-unreal#496]
### Environment

How do you use Sentry?
Sentry SaaS (sentry.io) 

Which version of the SDK? 0.15.1

How did you install the package? AssetStore

### Steps to Reproduce

1. Record a Session
2. Crash it

### Expected Result

Session appears in Sentry as crashed

### Actual Result

Session appears in Sentry, but not as crashed. Crash makes it into Sentry though .

>So I've tried setting this value Settings->EnableAutoSessionTracking = true; in USentrySettings . That didnt seem to do anythihng
And then I tried calling USentrySubsystem::StartSession();, which then does show up  as a session in the dashboard. But then when I attempt to crash from that session, it still counts as successful. I made sure not to call USentrySubsystem::EndSession()
And then I tried "hack" the Sentry library code in SentrySubsystemDesktop::HandleBeforeSend code to call sentry_end_session_with_status with a crash status defined in sentry.h. This didnt seem to work either because I think HandleBeforeSend doesnt seem to be called..

**Comment by gid-sentry** [comment]
Internal ticket here: https://getsentry.atlassian.net/browse/FEEDBACK-2108

**Comment by bruno-garcia** [comment]
@tustanivsky any idea what might be happening?

**Comment by bruno-garcia** [comment]
@tustanivsky is that somewhere on our docs? If not, we should add it

**Comment by strangezakary** [comment]
Hi we are experiencing this issue as well. We are on Unreal 5.3 and are getting errors with crash_type: crash and we are still seeing 100% crash-free. We are using automatic session tracking, any ideas what to look at?

**Comment by strangezakary** [comment]
Looking more into this it doesn't look like sentry_end_session_with_status is called anywhere so how would it know if it crashed?

**Comment by tustanivsky** [comment]
@strangezakary Do you have `Enable automatic crash capturing (Windows, UE 5.2+)` option turned on in Sentry plugins settings?

**Comment by strangezakary** [comment]
@tustanivsky We do. I think the default is on

**Comment by strangezakary** [comment]
Actually it looks like we don't? Do we need this on?

**Comment by tustanivsky** [comment]
Yes, this has to be enabled. Basically, it allows you to utilize Sentry's native integration for capturing crashes instead of the engine's default error-handling mechanism and thus track session status properly.

**Comment by tustanivsky** [comment]
Closing this issue as it seems to be related to a current limitation of the plugin regarding the [Release & Health](https://docs.sentry.io/platforms/unreal/configuration/releases/) feature:
```
On Windows, release health support for Unreal Engine is currently limited to games made with UE 5.2 or later,
with the option "Enable automatic crash capturing (Windows, UE 5.2+)" in the plugin settings enabled.
```
